_posts/02-02-2017-Applecheesecake.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  "Apple Cheesecake"
+date:   2017-02-02
+categories: main
+---
+
+![food](http://images-gmi-pmc.edge-generalmills.com/357f1106-e2a4-4df6-8543-2abdc043948a.jpg)
+
+## Ingredients
+1  pouch (1 lb 1.5 oz) Betty Crocker™ sugar cookie mix
+1 1/2 teaspoons ground cinnamon
+1/2 cup butter, softened
+1 tablespoon butter, melted
+1 package (8 oz) cream cheese, softened
+1/4 cup sugar
+1 tablespoon Gold Medal™ all-purpose flour
+1 teaspoon vanilla
+1 egg
+1 1/4 cups apple pie filling with more fruit, chopped (from 21-oz can)
+1/2 teaspoon ground cinnamon
+
+## Directions
+1. Heat oven to 350°F. Grease 9-inch glass pie plate with shortening.
+2 In large bowl, stir together cookie mix and 1 1/2 teaspoons cinnamon. Add 1/2 cup softened butter; mix with pastry blender or fork until crumbly. Press half (about 2 cups) of the crumbly mixture in bottom and up side of pie plate. Bake 10 minutes.
```

Comment:
These numbers aren't looking quite right. Maybe try adding a `.` after the digit?